### PR TITLE
Create calibre-web.subdomain.conf.sample

### PR DESCRIPTION
**Pull request by drizuid**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

**Comment by drizuid** [comment]
submitting as the last PR was closed without the required modifications for addition

File: calibre-web.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		#auth_request /auth;
+		#error_page 401 =200 /login;
+
+    include /config/nginx/proxy.conf;
+		resolver 127.0.0.11 valid=30s;
+		set $upstream_calibre_web calibre-web;
+		proxy_pass http://$upstream_calibre_web:8083;
+		proxy_set_header Host $http_host;
+		proxy_set_header X-Scheme $scheme;
+		proxy_set_header X-Script-Name /calibre;
```
Comment (aptalca) [inline calibre-web.subdomain.conf.sample]: is this line accurate for a subdomain proxy conf?

Comment (drizuid) [reply calibre-web.subdomain.conf.sample]: not 100% sure, i pulled it from the sample and have been using it as is for about 6 months without thought. I just tested with it removed and it still worked fine.

Comment (aptalca) [reply calibre-web.subdomain.conf.sample]: Can you remove it? Then I'll merge.

Thanks

**Pull request merged**